pytorch_toolkit/instance_segmentation_2/model_templates/coco-instance-segmentation/instance-segmentation-0002/template.yaml

```diff
@@ -0,0 +1,62 @@
+name: instance-segmentation-0002
+domain: Instance Segmentation 2
+problem: COCO Instance Segmentation
+framework: OTEDetection
```

Comment:
please set framework the same as in object detection models

Reply:
Fixed.